test/suites/smoke-test-dancelight/test-external-validators-rewards.ts

```diff
@@ -82,13 +82,16 @@ describeSuite({
 
                 // The event is triggered, nonce should be incremented
                 if (event) {
-                    expect(nonceDiff).toEqual(1);
-                    expect(
-                        sovereignBalanceCheckpointA.toBigInt() - sovereignBalanceCheckpointB.toBigInt()
-                    ).to.be.lessThan(0);
+                    const supplyBefore = (await apiAtCheckpointA.query.balances.totalIssuance()).toBigInt();
+                    const sovereignIssuance = (supplyBefore * 32641n) / 1_000_000_000n;
```

Comment:
can we get this value through a constant? I am asking just in case we change it in the future, to make it be reuse in both starlight and dancelight